prow/cmd/line/main.go

```diff
@@ -510,7 +510,9 @@ func (c *testClient) formatFailureComment(url string) string {
 	}
 
 	// The deletion logic requires that it start with context.
-	bodyFormat := `%s [**failed**](%s) for commit %s. [Full PR test history](http://pr-test.k8s.io/%s). cc @%s
+	bodyFormat := `%s [**failed**](%s) for commit %s. [Full PR test history](http://pr-test.k8s.io/%s). 
+
+cc @%s, [your PR dashboard](https://k8s-gubernator.appspot.com/pr)
```

Comment:
That seems kind of out of place. But OK, I don't know what is the right place :-)

Who is this talking to though? it reads "cc @spxtr, your dashboard", and links to `/pr`, which is *my* dashboard.
That's kind of confusing. Just my 2 cents =)